tests/tests/src/preserve_jsx_test.mjs

```diff
@@ -0,0 +1,69 @@
+import { Fragment, jsx, jsxs } from "react/jsx-runtime";
+import * as Primitive_option from "rescript/lib/es6/Primitive_option.js";
+import * as JsxRuntime from "react/jsx-runtime";
+let React = {};
+let ReactDOM = {};
+function Preserve_jsx_test$Icon(props) {
+  return /* @__PURE__ */ jsx("strong", {});
+}
+let Icon = {
+  make: Preserve_jsx_test$Icon
+};
+let _single_element_child = /* @__PURE__ */ jsx("div", { children: /* @__PURE__ */ jsx("h1", { children: "Hello, world!" }) });
```

Comment:
this file is not in preserve mode?

Reply:
Because it runs in the mocha tests (the glob picks it).
And because it does that, it needs to be transpiled first.

Reply:
Would be nice to see the output. Like this it's pretty hard to figure out if something is not quite right.
I'd take the preserved output over a mocha test if there's a choice,.

Reply:
Yeah, that could work. Was just trying to play ball with the existing stuff.